src/Components/Classes/index.jsx

```diff
@@ -41,72 +31,35 @@ function Projects() {
       const data = await response.json();
       setSelectedClass(data.data);
     } catch (error) {
-      setResponseModal({ error: true, msg: error });
-      setShowModal(true);
-      throw new Error(error);
-    }
-  };
-
-  const createClass = async (newClass) => {
-    newClass.trainer = [newClass.trainer];
-    const bodyClasses = {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-      body: JSON.stringify(newClass)
-    };
-    try {
-      const response = await fetch(`${process.env.REACT_APP_API_URL}/api/class/`, bodyClasses);
-      const data = await response.json();
-      if (data.length !== 0 && !data.error) {
-        setClasses([
-          ...classes,
-          {
-            _id: data._id,
-            activity: {
-              name: data.activity.name
-            },
-            trainer: [{ firstName: data.trainer[0].firstName }],
-            day: data.day,
-            time: data.time,
-            capacity: data.capacity
-          }
-        ]);
-        setResponseModal({ error: false, msg: 'Class created sucessfully' });
-        setShowModal(true);
-      } else {
-        setResponseModal({ error: true, msg: data.message });
-        setShowModal(true);
-      }
-    } catch (error) {
-      setResponseModal({ error: true, msg: error });
-      setShowModal(true);
-      throw new Error(error);
+      setShowModal({ show: true, state: 'fail', msg: 'Something went wrong :( try again later' });
     }
   };
 
   return (
     <section className={styles.container}>
       <h2>Class List</h2>
       <div>
-        <ClassList
-          classes={classes && classes}
-          getById={getById}
-          selectedClass={selectedClass}
-          setRenderData={setRenderData}
-          setResponseModal={setResponseModal}
-          setShowModal={setShowModal}
-          trainers={trainers}
-          activities={activities}
-        ></ClassList>
-        <Form createClass={createClass} trainers={trainers} activities={activities}></Form>
-        <Modal
-          showModal={showModal}
-          responseModal={responseModal}
-          onClose={() => setShowModal(!showModal)}
-        ></Modal>
+        {classes.length !== 0 ? (
+          <ClassList
+            classes={classes && classes}
+            getById={getById}
+            selectedClass={selectedClass}
+            setRenderData={setRenderData}
+          ></ClassList>
+        ) : (
+          'Something went wrong :( try again later'
```

Comment:
this should be  'There are not classes yet. Add new ones!!'  like in the correction of my pr